modules/studio-service/src/main/java/org/opencastproject/studio/endpoint/StudioEndpoint.java

```diff
@@ -0,0 +1,146 @@
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/")
+@RestService(name = "seriesservice", title = "Series Service",
+    abstractText = "This service creates, edits and retrieves and helps managing series.", notes = {
+    "All paths above are relative to the REST endpoint base (something like http://your.server/files)",
+    "If the service is down or not working it will return a status 503, this means the the "
+        + "underlying service is not working and is either restarting or has failed",
+    "A status code 500 means a general failure has occurred which is not recoverable and was "
+        + "not anticipated. In other words, there is a bug! You should file an error report "
+        + "with your server logs from the time when the error occurred: "
+        + "<a href=\"https://github.com/opencast/opencast/issues\">Opencast Issue Tracker</a>" })
+@Component(immediate = true, service = StudioEndpoint.class, property = { "service.description=Studio REST Endpoint",
+    "opencast.service.type=org.opencastproject.studio", "opencast.service.path=/studio" })
+public class StudioEndpoint {
+
+  private static final Logger logger = LoggerFactory.getLogger(StudioEndpoint.class);
+
+  private final Gson gson;
+
+  private ElasticsearchIndex elasticsearchIndex;
+  private SecurityService securityService;
+
+  {
+    gson = new GsonBuilder().serializeNulls().create();
+  }
+
+  @GET
+  @Path("/series.json")
+  @Produces(MediaType.APPLICATION_JSON)
+  @RestQuery(name = "getSeries", description = "Returns series where the current user has write permissions.",
+      returnDescription = "A list of JSON series objects",
+      restParameters = {
+         @RestParameter(name = "filter", isRequired = false,
+          description = "Usage <Filter Name>:<Value to Filter With>. Filters can combine using a comma \",\"."
+            + " Available Filters: title, textFilter.", type = STRING),
+      },
+      responses = {
+      @RestResponse(description = "Returns a list of series.",
+          responseCode = HttpServletResponse.SC_OK)
+  })
+  public Response getSeries(@QueryParam("filter") String filter) {
+    SeriesSearchQuery query = new SeriesSearchQuery(securityService.getOrganization().getId(),
+        securityService.getUser());
+    query.withoutActions();
+    query.withAction(Permissions.Action.WRITE);
+
+    if (filter != null && !filter.isBlank()) {
+      for (String f : filter.split(",")) {
+        String[] filterTuple = f.split(":",2);
```

Comment:
```suggestion
        String[] filterTuple = f.split(":", 2);
```

Weird, the style checker should catch these things :thinking: